Bowon/MemberList/MemberList/MemberListViewController.swift

```diff
@@ -0,0 +1,172 @@
+    Member(memberImage: "임꺽정.png",memberNumber: "1", memberName: "임꺽정", memberAge: "20", memberPhoneNumber: "010-2345-2634", memberAddress: "서울특별시"),
+    Member(memberImage: "홍길동.png",memberNumber: "2", memberName: "홍길동", memberAge: "20", memberPhoneNumber: "010-5645-8678", memberAddress: "광주광역시")
+]
+
+protocol MemberDetailDelegate {
+    func didUpdateMember(_ member: Member)
+}
+
+class MemberListViewController:UIViewController{
+    var selectedMemberIndex: Int = 0
+    
+    private lazy var myTableView:UITableView = {
+        let tableView = UITableView()
+        tableView.dataSource = self
+        tableView.delegate = self
+        return tableView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        setConstraint()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        setNavigationButton()
+        myTableView.reloadData()
+    }
+    
+// MARK: - constraint
+    private func setConstraint(){
+        let safeArea = view.safeAreaLayoutGuide
+
+        myTableView.register(MemberListViewCell.self, forCellReuseIdentifier: MemberListViewCell.identifier)
+
+        self.view.addSubview(myTableView)
+        
+        myTableView.translatesAutoresizingMaskIntoConstraints = false
+                
+        myTableView.rowHeight = UITableView.automaticDimension
+        myTableView.estimatedRowHeight = UITableView.automaticDimension
+
+        NSLayoutConstraint.activate([
+            myTableView.topAnchor.constraint(equalTo: safeArea.topAnchor),
+            myTableView.bottomAnchor.constraint(equalTo: safeArea.bottomAnchor),
+            myTableView.leadingAnchor.constraint(equalTo: safeArea.leadingAnchor),
+            myTableView.trailingAnchor.constraint(equalTo: safeArea.trailingAnchor)
+        ])
+    }
+    
+    private func setNavigationButton(){
+        let addButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(onTapAddBarButton))
+        self.navigationItem.rightBarButtonItem = addButton
+        self.navigationItem.leftBarButtonItem = self.editButtonItem
+        self.navigationItem.title = "회원 목록"
+    }
+    
+    @objc func onTapAddBarButton(_:UIBarButtonItem){
+        let AddMemberVC = AddMemberViewController()
+        self.navigationController?.pushViewController(AddMemberVC, animated: true)
+    }
+}
+
+// MARK: - extension UITableViewDelegate,UITableViewDataSource
+extension MemberListViewController: UITableViewDelegate, UITableViewDataSource, MemberDetailDelegate{
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: MemberListViewCell.identifier, for: indexPath) as? MemberListViewCell else { return UITableViewCell() }
```

Comment:
이런 한 줄에 많은 코드가 적혀있는 것들은 적절하게 인덴트를 맞춰주는게 좋아! 
```suggestion
        guard let cell = tableView.dequeueReusableCell(
          withIdentifier: MemberListViewCell.identifier,
          for: indexPath
        ) as? MemberListViewCell else { 
           return UITableViewCell()
         }
```